.eslintrc.json

```diff
@@ -19,7 +19,8 @@
     "react/react-in-jsx-scope": "off",
     "import/no-unresolved": "off",
     "no-shadow": "off",
-    "arrow-parens": ["error", "as-needed"]
+    "arrow-parens": ["error", "as-needed"],
+    "linebreak-style":"off"
   },
   "ignorePatterns": [
     "dist/",
```

Comment:
- [ ] OPTIONAL - This is an important requirement but since it wasn't mentioned in the previous review it's been marked as optional. Please ensure you use the same eslint config rules shared by other students. Ensure the rules match only those in this [link](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json).

Reply:
I will correct this in the next milestone